src/providers/twitch/api/Helix.cpp

```diff
@@ -578,6 +578,48 @@ void Helix::unblockUser(QString targetUserId,
         .execute();
 }
 
+void Helix::updateChannel(QString broadcasterId, QString gameId,
```

Comment:
Could you update the `src/providers/twitch/api/README.md` with the information about this request, including which scope is required?

Reply:
preferably it should go right below the `Get Channel` entry on line 99